src/services/GLSL/utilities.glsl

```diff
@@ -75,6 +75,9 @@ vec2 controlMapLookup(sampler2D controlMapTexture, vec2 pos, vec2 controlMapSize
     vec2 range = controlMapMax - controlMapMin;
     vec2 shiftedPoint = pos - controlMapMin;
     vec2 index = shiftedPoint / range * controlMapSize;
+    if (controlMapMax == controlMapMin) {
```

Comment:
I tried
```if (range.x == 0 && range.y == 0) { index = vec2(0, 0) };```
but it didn't seem to work. I am not very familiar with the syntax of GLSL, so please advice if there is a better solution.

Reply:
The current solution works fine. No regression is observed from e2e tests. If there are better solutions from @YuHsuan-Hwang , I will re-do the test.

Reply:
@crocka Does `if (range.x == 0.0 && range.y == 0.0) { index = vec2(0, 0) };` works?
`controlMapMax == controlMapMin` is also fine.

Reply:
This is actually the first thing I tried. It didn't work and it is giving me the following error.
![Screenshot from 2023-01-03 20-07-20](https://user-images.githubusercontent.com/94193689/210466207-300f8d83-f491-461f-af95-f00872460d66.png)

Reply:
Did you use `0.0` or `0`? You can check the console for error messages.

Reply:
I see the problem. We should use range.x == 0.0 && range.y == 0.0. I tried `index = vec2(0.0, 0.0) but ignored the if-statement.